src/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  max-width: 300px;
+  margin: 96px auto;
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 18px;
+  font-weight: 600;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+}
+
+.checkbox {
+  margin: 0 12px;
+}
+
+.index {
+  margin-left: auto;
+  margin-right: 12px;
+}
+
+.border {
+  border: 1px solid gray;
+}
+
+.input {
+  width: 282px;
+  height: 60px;
+  border: 1px solid gray;
+  padding: 0 0 0 16px;
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 18px;
+  font-weight: 600;
```

Comment:
It is a good to avoid the repetition of styles since you have these styles in the body already. Optionally, you can put these style properties in the universal selector if you are sure that it is going to run through the entire project. This will make your code DRY 👍

Reply:
.+1